test/cli/main.js

```diff
@@ -107,6 +107,12 @@ QUnit.module( "CLI Main", () => {
 		}
 	} );
 
+	QUnit.test( "exit code is 0 when no tests are run and failOnZeroTests is `false`", async assert => {
+		assert.expect( 0 );
+		const command = "qunit assert-expect/no-tests.js";
```

Comment:
I think we should also validate the TAP output as for other tests since this is a new use case for the formatter and I can see how that could cause a bug somewhere.

Reply:
@Krinkle there already is an existing test for `No tests were run`: https://github.com/qunitjs/qunit/blob/main/test/cli/main.js#L99. Do you think this is sufficient?

Reply:
This would be similar, but produce a success instead of a failure. Observing what an empty success looks like, I think, would be interesting and useful to assert. E.g. dealing with zero in these places correctly.

Compare to https://github.com/qunitjs/qunit/blob/2.15.0/test/cli/fixtures/expected/tap-outputs.js#L148-L168

Reply:
Ah, fair enough. Yeah, makes sense, let me update